### ISSUE TITLE
Sometimes fails with `iPhoneSimulator: Timed out waiting...`

### ISSUE THREAD

**Issue by viktorbenei** [bitrise-io/steps-xcode-test#4]
```
xcodebuild[964:4881]  iPhoneSimulator: Timed out waiting 120 seconds for simulator to boot, current state is 1.
```

This is a known issue which can happen anywhere, but it happens more frequently in performance limited environment, for example running the Simulator inside a Virtual Machine.

**Comment by viktorbenei** [comment]
A second run always succeeds, we'll add a fix to this step, similar to this fix: https://github.com/KrauseFx/fastlane/blob/ca11b65477bcf14f96d4c6376b6924ac466f2ecc/lib/fastlane/actions/xcodebuild.rb#L208

**Comment by tomaskraina** [comment]
Thanks @viktorbenei 

so I can just use fastlane that has this issue covered, right?

**Comment by viktorbenei** [comment]
@tomaskraina sure! The PR was actually done by us, and we'll release the same fix for our Xcode Test step in (hopefully) less than an hour.

**Comment by tomaskraina** [comment]
awesome! @viktorbenei

**Comment by viktorbenei** [comment]
fixed in https://github.com/bitrise-io/steps-xcode-test/pull/5 - starting the final tests of the fix

**Comment by viktorbenei** [comment]
@tomaskraina `v1.1.0` was just released, it now includes this fix

**Comment by viktorbenei** [comment]
Let us know if you still experience the issue with `1.1.0`. I'll close this issue for now.

**Comment by viktorbenei** [comment]
Fix is not correct, it'll always fail to start it a second time. New fix will be released asap, in the next hours.

**Comment by viktorbenei** [comment]
Fixed in `v1.1.2`
